packages/core/strapi/package.json

```diff
@@ -12,13 +12,25 @@
   "bin": {
     "strapi": "./bin/strapi.js"
   },
+  "strapi": {
+    "internalPlugins": [
+      "@strapi/plugin-content-manager",
```

Comment:
Could we use a constant in the code ? this won't be configurable once released maybe this will be easier to read in the code :)